bower.json

```diff
@@ -1,17 +1,16 @@
 {
   "name": "ember-stripe-service",
   "dependencies": {
-    "handlebars": "~1.3.0",
     "jquery": "^1.11.1",
-    "ember": "1.8.1",
-    "ember-data": "1.0.0-beta.12",
-    "ember-resolver": "~0.1.11",
-    "loader.js": "ember-cli/loader.js#1.0.1",
+    "ember": "1.10.0",
+    "ember-data": "1.0.0-beta.15",
```

Comment:
I don't think you need ember-data for this addon, so it would be even better to not have it listed as a dependency.